oam/etc/Columnstore.xml

```diff
@@ -249,10 +249,6 @@
 		<ExternalCriticalThreshold>90</ExternalCriticalThreshold>
 		<ExternalMajorThreshold>80</ExternalMajorThreshold>
 		<ExternalMinorThreshold>70</ExternalMinorThreshold>
-		<!-- <TempDiskPath>/tmp</TempDiskPath>
```

Comment:
I think we must leave a separate setting for a temporary directory so that it can point to a faster storage. This affects the startup procedures.